account_financial_report/report/abstract_report_xlsx.py

```diff
@@ -297,7 +297,9 @@ def write_line_from_dict(self, line_dict, report_data):
                     value or "",
                     report_data["formats"]["format_right"],
                 )
-        report_data["row_pos"] += 1
+            else:
+                self.write_non_standard_column(cell_type, col_pos, value)
+        self.row_pos += 1
```

Comment:
Why changing this? It seems incorrect

Reply:
In my use case. I am adding the purchase order information to the report. When I query the purchase_id field in the method "_get_report_columns" I do it this way:

```

            6: {
                "header": _("Purchase Order"),
                "field": "purchase_id",
                "width": 11,
                "type": "purchase",
            },
```

With that hook I can then write the value I need. Putting type "many2one" in the columns declaration was not working in my case (at least in v13). Then I decided to put the hook so I can take the value when printing the line. I will check in this version if it works without the hook.

Reply:
I was just talking about changing `report_data["row_pos"] += 1` with `self.row_pos += 1`

Reply:
I overlooked this. Yes, that is non intentional and seems an incorrect merge conflict resolution.

Reply:
Fixed and it is working now. Sorry all.